cmd/vic-machine/create/create.go

```diff
@@ -27,6 +27,7 @@ import (
 	"github.com/vmware/vic/cmd/vic-machine/data"
 	"github.com/vmware/vic/cmd/vic-machine/validate"
 	"github.com/vmware/vic/lib/install/management"
+	"github.com/vmware/vic/pkg/certificate"
```

Comment:
yeah it's not vSphere specific code though, it's a general certificate utility package